enso/experiment/crfac.py

```diff
@@ -0,0 +1,70 @@
+"""Module for any LR-style experiment."""
+from sklearn.linear_model import LogisticRegression
+import sklearn_crfsuite
+import numpy as np
+import pandas as pd
+import spacy
+import crf_processing
```

Comment:
Doesn't seem like you committed this file

Reply:
just pushed it. Yeah, I also probably want to factor that out in some better way.